File: Rotate Array.cpp
class Solution {
public:
    void rotate(vector<int>& nums, int k) {
        vector<int> ans;
        k=k%nums.size();
        for(int i=0;i<nums.size();i++)
        {
            ans.push_back(nums[(nums.size()-k+i)%(nums.size())]);
        }
        nums=ans;
    }
};

File: Rotate Array.py
class Solution:
    def rotate(self, nums: List[int], k: int) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        k=k%len(nums)
        for i in range(k):
            nums.insert(i,nums[len(nums)-k+i])
        for i in range(k):
            nums.pop()
